backend/contextProperties.js

```diff
@@ -56,6 +56,21 @@ const setResponsePagination = (context, headers) => {
   context.responseHeaders = copyNamedHeaders(headerNames, (headers && normalizeHeaderNames(headers)) || {})
 }
 
+const getPaginationForPageRequest = ({ requestHeaders }) => {
+  if (!requestHeaders) return { page: 0, size: 20 }
+
+  const pageOffset = requestHeaders['page-offset']
+  const size = requestHeaders['page-limit'] || 20
```

Comment:
have to translate between the way that prison api takes the pagination parameters and prisoner offender search